src/main/java/botbot/Ui.java

```diff
@@ -0,0 +1,58 @@
+package botbot;
+
+import java.util.Scanner;
+
+/**
+ * Represents the UI of the chatbot.
+ */
+public class Ui {
```

Comment:
Perhaps, this class could be named "UI" instead of "Ui", since it is an acronym where both letters represent different words.

> Class/enum names must be nouns and written in PascalCase. - Java coding standard @ [SE-EDU/guides](https://se-education.org/guides/conventions/java/intermediate.html#guides-for-se-student-projects)